src/js/pages/Topic.js

```diff
@@ -2,41 +2,50 @@ import React from 'react';
 import { withRouteData } from 'react-static';
 import { get } from 'lodash';
 import { cleanServiceLinks } from 'js/helpers/cleanData';
+import PropTypes from 'prop-types';
```

Comment:
👏 Should we talk about adding propTypes for next sprint? Seems like a good house-cleaning/code-usability thing like storybooks.

Reply:
yup, let's commit to adding them into components as we work on them. Leave it better than you found it, right?